### DISCUSSION TITLE
New install giving Firebase compatibility errors

### DISCUSSION THREAD

**Discussion by 34fame** [robingenz/capacitor-firebase-authentication#143] [Q&A]
Added `@robingenz/capacitor-firebase-authentication` and `firebase` packages.  Ran `npx cap update`.

@robingenz/capacitor-firebase-authentication: 0.4.3
firebase: 9.6.8

<img width="1732" alt="Screen Shot 2022-03-14 at 2 13 18 PM" src="https://user-images.githubusercontent.com/49797484/158244384-45662d42-0c21-45eb-bdbb-83721e9298d4.png">

**Comment by 34fame** [comment]
Forcing 0.4.1 & Firebase 9.6.5 prevents the error.

**Comment by robingenz** [comment]
Duplicate of #141

Please run `cd /ios/App/ && pod install --repo-update` and try it again.
